focus-ios/focus-ios-tests/XCUITest/DragAndDropTest.swift

```diff
@@ -14,10 +14,15 @@ class DragAndDropTest: BaseTestCase {
         waitForWebPageLoad()
 
         // Check the text in the search field before dragging and dropping the url text field
-        waitForExistence(app.webViews.textFields[websiteWithSearchField["urlSearchField"]!])
-        // DragAndDrop the url for only one second so that the TP menu is not shown and the search box is not covered
-        urlBarTextField.firstMatch.press(forDuration: 1, thenDragTo: app.webViews.otherElements["search"].firstMatch)
-        // Verify that the text in the search field is the same as the text in the url text field
-        XCTAssertEqual(app.webViews.textFields[websiteWithSearchField["urlSearchField"]!].firstMatch.value as? String, websiteWithSearchField["url"]!)
+        if #unavailable(iOS 17) {
+            waitForExistence(app.webViews.otherElements[websiteWithSearchField["urlSearchField"]!])
+            urlBarTextField.firstMatch.press(forDuration: 1, thenDragTo: app.webViews.otherElements["search"].firstMatch)
```

Comment:
I could not verify the presence of the URL in the search box for now. 🤷🏼‍♀️

Reply:
So the URL could not be validated for 15 and 16? I want to make sure i understand that the validation is still performed for iOS 17.

Reply:
`print(app.debugDescription)` gives me the following on iOS 16.4. Note that there's a line break after `No document titles found`.
```
                                  Other, 0x122715010, {{15.0, 466.0}, {359.0, 175.0}}, label: 'search'
                                    Other, 0x122715130, {{15.0, 466.0}, {1.0, 1.0}}, label: 'Search MDN'
                                      StaticText, 0x122715250, {{15.0, 470.0}, {96.0, 20.0}}, label: 'Search MDN', value: Search MDN
                                    Other, 0x122715370, {{16.0, 467.0}, {358.0, 66.0}}, label: 'Search MDN', placeholderValue: '   ', value: https://developer...., Focused
                                    Button, 0x122715490, {{326.0, 467.0}, {24.0, 64.0}}, label: 'SEARCH'
                                    Other, 0x12270f650, {{40.0, 534.0}, {310.0, 107.0}}, label: 'Search MDN'
                                      Other, 0x12270f770, {{41.0, 535.0}, {308.0, 105.0}}, label: 'No document titles found.
Site search for https://developer.mozilla.org/en-US/', Selected
```